modules/ROOT/pages/import-api-def-dc.adoc

```diff
@@ -1,35 +1,39 @@
-= To Import an API Definition from Design Center
+= Importing an API Definition from Design Center
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 Anypoint Studio allows you to import an API Definition from Design Center and implement it either into a new Mule Project, or an existing one.
 
-== To Implement an API imported from Design Center Into a New Project
+== Implementing an API imported from Design Center Into a New Project
 
-. Click File > New > Mule Project to create a project, and set the field values in the project wizard:
+. Select *File* > *New* > *Mule Project* to create a project, and set the field values in the project wizard:
 +
-* Type a name for your project.
-* Select a Mule runtime version.
-* Check Add APIkit Components.
-* In API Definition, click the "..." button to and select Design Center.
+.. Type a name for your project.
+.. Select a Mule runtime version.
+.. Select *Specify API definition file location or URL*.
+.. To the right of the *Location* field, click the "..." button and select *Design Center*.
 . In the Browse Design Center for APIs window:
 * Select your Anypoint Platform user and business group.
 +
-Additionally you can click the "Add Account" button to use a new Anypoint Platform account.
-. From the list of available APIs, select the one you want and click OK.
-. Click Finish.
+Logging previously into Anypoint Platform, makes your Username and Business Group details grayed out.
```

Comment:
Would you explain to me what you mean in this sentence?

Reply:
Yes. This sentence is like a "clarification note" for the user since in step 2 of the first process says, "Select your Anypoint Platform user and business group". However, the field with the Username and Business Group ID which already contain the user details are read-only (grayed out) meaning that the user had to log into Anypoint Platform before the dialog with this field appeared.
<img width="812" alt="Screen Shot 2019-04-04 at 2 55 24 PM" src="https://user-images.githubusercontent.com/47541849/55583844-41112980-56f9-11e9-8cfc-83c25261017f.png">

Reply:
I didn't want to use passive voice. Do you think I should change it to "If you previously logged into Anypoint Platform, your Username and Business Group details are grayed out."? Let me know if you if you think anything else that could help. Thanks.

Reply:
What does "previously" mean here? Any time in the past, including, say, one week ago?

Reply:
I changed the sentence to If you are logged into Anypoint Platform, your Username and Business Group will show grayed out.

Reply:
"Will show" isn't quite right, either. "Will be" is best. Yes, it's passive; however, it's acceptable in many cases in tech writing.